cmd/restic/global.go

```diff
@@ -105,6 +107,13 @@ func init() {
 		return nil
 	})
 
+	//set MinPackSize to 16 if not set in env
+	minPackSize, err := strconv.ParseUint(os.Getenv("RESTIC_MIN_PACKSIZE"), 10, 32)
```

Comment:
My intention was to initially only increase the default pack size to 16 MB, but not provide an option for now. Sorry, if my comment in #2750 wasn't clear enough.

Reply:
surely is a more suitable default than current one, but without the option the whole change loses half of its usefulness (IMHO, of course).

Reply:
16 MB sounds small too. I was looking forward to setting it to e.g. 128MB or 256MB, as I backup up terabytes and having hundreds of thousands or millions of packfiles is not really great on spinning rust.

Reply:
same for me, I aim to 128/256 pack size, but I agree that this could not be a sane default for everybody.
the option to tune this parameter should be present.

Reply:
Ok, I give in ^^ . Let's keep the parameter, but limit the allowed range for now to 4-128MB with a default of 16MB. Eventually I'd like to change the default to `auto` assuming that we find a way to automatically determine a reasonable pack size that works for most use cases. I'll add some further rationale below.